dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/manager/internal/GradleProjectInspector.java

```diff
@@ -129,8 +126,21 @@ public File getLibertyBuildPluginConfigFile(IProgressMonitor mon) throws CoreExc
         // Note the Gradle build can be configurable.  The default value is 'build'.
         IFile f = project.getFile(LibertyGradleConstants.LIBERTY_PLUGIN_CONFIG_PATH);
         // Check the if the Gradle build has the file.
+        
+        if (!f.exists()) {
+           	Trace.trace(Trace.INFO, "The gradle project " + project.getName() + " does not have the liberty plugin config path.  Liberty Tools will not recognize this as a Liberty Gradle project.");
+        }
+        
         if (f != null && f.exists())
             return f.getLocation().toFile();
+        
+        IFile buildGradleFile = project.getFile("build.gradle");
+        if (buildGradleFile != null && !buildGradleFile.exists()) {
```

Comment:
Will this work if the build file is not called build.gradle?  The user can change the name of the build file by creating a settings.gradle file with:
rootProject.buildFileName = 'myBuildFile.gradle'
If I import a project now that has a different build file name it works - creates the runtime and server.  Will it still work after this change?

Reply:
This will mean we have to parse this groovy file - it's not as simple as parsing a properties file with name/value pairs.